src/Table.jsx

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import allCountryScores from "./scores";
+import "./Table.css"
+
+function Table() {
+  return (
+    <>
+      <h1>High score per Country</h1>
+      <div>
+        {allCountryScores.map(({ name, scores }, index) => {
+          return (
+            <div key={index}>
+              <table>
+                <thead>
+                  <tr>
+                    <th scope="col">{name}</th>
+                  </tr>
+                </thead>
+                <tbody>
+                  {scores.map(({ n, s }, index) => {
```

Comment:
Where we have a nested `map`, I would consider a separate component. As a rule of thumb, if we have another loop inside of a component, it's probably a good opportunity to refactor into multiple components

Reply:
Sounds right, I will try to do this in future.